src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Title from 'Assets/Elements/Title';
+import CustomizedTextField from 'Assets/Elements/CustomizedTextField';
+import { Grid } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { StyledFormGroup } from 'Assets/Elements/StyledFormGroup';
+import { StyledButton } from 'Assets/Elements/StyledButton';
+import { StyledDivider } from '../Style';
+import { useFormContext } from 'react-hook-form';
+const RecommendationsOnRelease = () => {
+  const { t } = useTranslation();
+  const {
+    permission,
+    register,
+    watch,
+    setPopUpProps,
+    setValue,
+  } = useFormContext();
+  const instructionsForFurtherTreatment = watch(
+    'instructionsForFurtherTreatment',
+  );
+  const callBack = (data, name) => {
+    setValue(name, data);
+  };
+
+  const handlePopUpProps = (title, fields, id, callBack, name) => {
+    setPopUpProps((prevState) => {
+      return {
+        ...prevState,
+        popupOpen: true,
+        formFieldsTitle: title,
+        formFields: fields,
+        formID: id,
+        setTemplatesTextReturned: callBack,
+        name,
+      };
+    });
+  };
+  return (
+    <StyledFormGroup>
+      <Title
+        label={t('Recommendations on release')}
+        fontSize='22px'
+        bold
+        color='#000b40'
+      />
+      <StyledDivider />
+      <Grid
+        container
+        direction='row'
+        justify='flex-start'
+        alignItems='baseline'>
+        <CustomizedTextField
+          name='instructionsForFurtherTreatment'
+          inputRef={register}
+          label={t('Instructions for further treatment')}
```

Comment:
for all the inputs with "select template" button, better to create const that contains the title of the input for example - t('Instructions for further treatment')
and use the variable for the label and for the first parameter in the handlePopUpProps.

now we have the same string in two places-
```
 label={t('Instructions for further treatment')}

 handlePopUpProps(
              t('Instructions for further treatment'),
```

It's low issue but more professional